draft-ietf-jsonpath-base.md

```diff
@@ -320,19 +326,21 @@ $.store.book[?(@.price < 10)].title
-| `['name']`  | child selectors for JSON objects: [index selector](#index-selector)                                                                 |
-| `..name` <br> `..[3]` | descendants: [descendant selector](#descendant-selectors)                                                                     |
-| `.*` <br> `[*]` | all child member values and array elements: [dot wildcard selector](#wildcard), [index wildcard selector](#index-wildcard-selector) |
-| `[3]`       | [index (subscript) selector](#index-selector): index current node as an array (from 0)                                              |
-| `[..,..]`   | [list selector](#list-selector): allow combining selector styles                                                                    |
-| `[0:100:5]` | [array slice selector](#slice): start:end:step                                                                                      |
-| `?...`      | [filter selector](#filter-selector)                                                                                                 |
-| `()`        | expression: within [filter selectors](#filter-selector)                                                                            |
+| JSONPath            | Description                                                                                                             |
+|---------------------|-------------------------------------------------------------------------------------------------------------------------|
+| `$`                 | [root node identifier](#root-identifier)                                                                                |
+| `[<selectors>]`     | [child appender](#child-appender) selects zero or more children of JSON objects and arrays; contains one or more selectors, separated by commas        |
+| `..[<selectors>]`   | [descendant child appender](#descendant-appender): recursive version of the child appender                             |
+| `*`                 | [wildcard selector](#name-selector): selects all children of an array or object                                         |
+| `'name'`            | [name selector](#name-selector): selects a named child of an object                                                     |
+| `3`                 | [index selector](#index-selector): selects an indexed child of an array (from 0)                                        |
+| `0:100:5`           | [array slice selector](#slice): start:end:step for arrays                                                               |
+| `?<expr>`           | [filter selector](#filter-selector): selects particular children using a boolean expression                             |
+| `@`                 | [current node appender](#filter-selector) (valid only within filter selectors)                                          |
```

Comment:
`@` is a *starter. For consistency with `$`, it would be better to describe it as a "current node identifier".